docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_creating_custom_test.adoc

```diff
@@ -29,68 +29,75 @@ metadata:
   name: example-pipeline
 spec:
   params:
-  - name: username
-    default: world 
-  tasks: 
-----
-
-. In the `.pipeline.spec` path, declare a new task. For any valid task, the fields in the following example are all required, except `params`. 
-
-+
-Example task declaration:
-
-+
-[source]
-----
-tasks:
-  - name: task-1
-    params:
     - name: username
-      value: $(params.username)
-    taskSpec:
+      default: world 
+  tasks: 
+    - name: task-1
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/**/**-examples
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: tasks/task-1.yaml
       params:
-      - name: username
-      steps:
-      - image: alpine:3.15
-        script: |
-          echo "hello $(params.username)"
-
+        - name: username
+          value: $(params.username)
```

Comment:
Why this change? 
If you want to fix unrelated parts to injected variables, open a new PR.

I discontinue review, I'm not going to review PR with changing scope

Reply:
It's mentioned in the A/C item -  Examples and usage for docs team (file a story), i.e. how user can create working integration test to test deployed services are provided. So you meant to file a new JIRA ticket for it ?

Reply:
But using those options we are documenting, SNAPSHOT param

if all examples are completelly wrong, then we should have a new ticket

Reply:
what I want to achieve here is, that it's clear to users how to use those parameters for creating their own integration test tasks:
- user should be able to find in doc what is injected into test pipeline by appstudio/integration-service
- user should be able to clearly understand what the values are (mainly in structured data like SNAPSHOT)
- user should be able to understand how to easily use params, and how to integrate them into testing pipeline (one example is better than 1000 words)

So maybe we can have a subsection here, with just single example, how to construct task which can use SNAPSHOT (for example, task may just print image url)

Reply:
An example needs its context. With the original pipeline and task in the page, it's hard to add a "simple example" for the SNAPSHOT from my understand.

Reply:
ok then